src/components/Admin/Onboard/ProductAddModal.vue

```diff
@@ -204,7 +229,7 @@
       reorderProducts () {
         let productIDs = []
         for (var i = 0; i < this.current_category.products.length; i++) {
-          productIDs.push(this.current_category.products[i].asset_id)
+          productIDs.push(this.current_category.products[i].product_id)
```

Comment:
You need to send the product ids you'd like to reorder. Stores can have multiple items with the same image (think of stock images for example)